### DISCUSSION TITLE
What is improved in Fooocus 2.4.3 than Fooocus 2.4.1

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3453] [Q&A] [answered]
I would like to know the main difference between them.
Thank you.

**Comment by ashailrizvi** [comment]
https://github.com/lllyasviel/Fooocus/discussions/3437

**Comment by mashb1t** [comment, marked as answer]
See https://github.com/lllyasviel/Fooocus/releases / https://github.com/lllyasviel/Fooocus/blob/main/update_log.md
